Doc 1579/ttp step 2 guide pages (#563)
swan-io/docs: docs/preview/in-person/cards/guide-create-payments.mdx

---
title: Create in-person card payments
---

# Create in-person card payments

Create payment intents so your merchants can accept card payments from customers using their smartphone terminal.

## Prerequisites {#prerequisites}

- A **merchant profile** with the status **Enabled**.
- An **in-person card payment method** with the status **Enabled**. If you don't have one yet, [request the payment method](./guide-request-method.mdx) first.
- A project access token, or a user access token with **Can manage members** rights.
- The [Stripe Terminal SDK](https://docs.stripe.com/terminal/payments/setup-sdk) integrated in your terminal app.

## Step 1: Get a connection token {#connection-token}

> Before the terminal app can discover and connect to a reader, it needs a connection token from Stripe.

Reviewer: ```suggestion
Get a connection token from Stripe so the terminal app can discover and connect to a reader
```
Frontloading the action.
